healthcheck/core/src/test/java/org/apache/felix/hc/core/impl/executor/async/cron/CronSchedulerTest.java

```diff
@@ -0,0 +1,114 @@
+package org.apache.felix.hc.core.impl.executor.async.cron;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.apache.felix.hc.core.impl.executor.async.cron.SampleCronJob1.Callback;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CronSchedulerTest extends Assert {
+
+    private static final Object SYNC_OBJECT = new Object();
+
+    @Test
+    public void checkSchedulerCheckSingleThread() throws InterruptedException {
+        try (final CronScheduler scheduler = new CronScheduler(Executors.newFixedThreadPool(1), 0, 1, TimeUnit.SECONDS, "T1")) {
+            final int nIterations = 3;
+
+            final SampleCronJob1 service = new SampleCronJob2(s -> {
+                System.out.println("Iteration: " + s.counter);
```

Comment:
Is it OK to write to console during tests in Apache Felix project?

Reply:
fixed